compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/util/NumericLiteralSupport.java

```diff
@@ -68,6 +68,24 @@ public static boolean isHexLiteral(String numericLiteral) {
         return false;
     }
 
+    /**
+     * Check input for decimal indicator.
+     *
+     * @param literalValue literal to check
+     * @return true if Hex prefix is present, false otherwise
+     */
+    public static boolean isHexIndicator(String literalValue) {
```

Comment:
Shall we rename this to 
```suggestion
    public static boolean hasHexIndicator(String literalValue) {
```

Reply:
Fixed